src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5535,6 +5535,7 @@ internal static class PSSnapInHelpers
 #if !UNIX
                 {"Disable-PSRemoting",                new SessionStateCmdletEntry("Disable-PSRemoting", typeof(DisablePSRemotingCommand), helpFile) },
                 {"Enable-PSRemoting",                 new SessionStateCmdletEntry("Enable-PSRemoting", typeof(EnablePSRemotingCommand), helpFile) },
+                {"Get-PSHostProcessInfo",             new SessionStateCmdletEntry("Get-PSHostProcessInfo", typeof(GetPSHostProcessInfoCommand), helpFile) },
```

Comment:
What about `Enter-PSHostProcess` and `Exit-PSHostProcess`? They are supposed to work with `Get-PSHostProcessInfo`. Maybe we should exclude them all? @PaulHigin thoughts?

Reply:
Yes, these cmdlets should be exluded as well.